PyInstaller/utils/misc.py

```diff
@@ -20,7 +20,7 @@
 import sys
 
 from PyInstaller import log as logging
-from PyInstaller.compat import is_unix, is_win, BYTECODE_MAGIC
+from PyInstaller.compat import is_unix, is_win, BYTECODE_MAGIC, is_py2
```

Comment:
Could you please make the imports relative? And do the same with other files as you are touching them?

Later this could ease renaming 'PyInstaller' top-level module name to lowercase 'pyinstaller'.